overlord/configstate/configcore/corecfg.go

```diff
@@ -50,13 +50,28 @@ func coreCfg(tr Conf, key string) (result string, err error) {
 	return fmt.Sprintf("%v", v), nil
 }
 
+func validateExperimentalSettings(tr Conf) error {
+	layoutsEnabled, err := coreCfg(tr, "experimental.layouts")
+	if err != nil {
+		return err
+	}
+	if layoutsEnabled != "" && layoutsEnabled != "true" && layoutsEnabled != "false" {
```

Comment:
Sort-of pattern matching style:
```
switch layoutsEnabled {
case "true", "false", "":
    return nil
default:
    return fmt.Errorf("experimental.layouts can only be set to 'true' or 'false'")
}
````
Feel free to ignore and not push another patch.